cassandra/cassandra.go

```diff
@@ -38,6 +39,13 @@ const (
 	pluginType = plugin.PublisherPluginType
 
 	serverAddrRuleKey          = "server"
+	portRuleKey                = "port"
+	timeoutRuleKey             = "timeout"
+	connectionTimeoutRuleKey   = "connectionTimeout"
+	initialHostLookupRuleKey   = "initialHostLookup"
+	ignorePeerAddrRuleKey      = "ignoreRuleKey"
+	createKeyspaceRuleKey      = "createKeyspace"
```

Comment:
My case is that cassandra administrator give me access to cassandra instance with prepared keyspace for me. I cannot create any other keyspace, so even attempt is killing publisher flow.